core-feature-pack/common/src/main/resources/content/bin/jboss-cli.sh

```diff
@@ -82,8 +82,15 @@ JAVA_OPTS="$JAVA_OPTS -Dcom.ibm.jsse2.overrideDefaultTLS=true"
 
 LOG_CONF=`echo $JAVA_OPTS | grep "logging.configuration"`
 if [ "x$LOG_CONF" = "x" ]; then
-    exec "$JAVA" $JAVA_OPTS -Dlogging.configuration=file:"$JBOSS_HOME"/bin/jboss-cli-logging.properties -jar "$JBOSS_HOME"/jboss-modules.jar -mp "${JBOSS_MODULEPATH}" org.jboss.as.cli "$@"
+    eval \"$JAVA\" $JAVA_OPTS \
```

Comment:
We are using exec due to the fix for https://issues.redhat.com/browse/JBEAP-8808.
We can't use eval there. At least in the general case.